jupyterlab_caip_optimizer/src/service/test-contsants.ts

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
File seems to be misspelled (and idk how to make file-level comments)

Reply:
I'll fix this as well